jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/cloudfunctions/IngressSettingRule.java

```diff
@@ -30,6 +30,9 @@ public PolicyResult execute(Map<String, String> ruleParam, Map<String, String> r
             logger.info(PacmanRuleConstants.MISSING_CONFIGURATION);
             throw new InvalidInputException(PacmanRuleConstants.MISSING_CONFIGURATION);
         }
+        if (!StringUtils.isNullOrEmpty(esUrl)) {
+            esUrl = esUrl + "/gcp_cloudfunction/_search";
+        }
```

Comment:
you don't have to check `esUrl` here - it won't be null or empty string ever because it's being checked on line 29.